android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -188,12 +188,21 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>(), R
       return false
     }
 
+    private fun updateBackgroundColor(backgroundColor: Int, borderRadius: Float, selectable: Drawable?) {
+      val colorDrawable = PaintDrawable(backgroundColor)
+
+      if (borderRadius != 0f) {
+        colorDrawable.setCornerRadius(borderRadius)
+      }
+
+      val layerDrawable = LayerDrawable(if (selectable != null) arrayOf(colorDrawable, selectable) else arrayOf(colorDrawable))
+      background = layerDrawable
+    }
+
     fun updateBackground() {
       if (!needBackgroundUpdate) {
         return
       }
-      needBackgroundUpdate = false
```

Comment:
I don't see a reason for removing this (and if there is one, it would basically always stay `true` so there would be no need for this variable to exist).

Reply:
Good catch, I must have deleted it by accident while playing with some `Log.d`.

Brought back in [9cc93c3](https://github.com/software-mansion/react-native-gesture-handler/pull/2642/commits/9cc93c3baeeef1c80d17176169069e9e14420cd4)